pandas/_libs/tslibs/meson.build

```diff
@@ -4,38 +4,27 @@ tslibs_sources = {
     'base': {'sources': ['base.pyx']},
     'ccalendar': {'sources': ['ccalendar.pyx']},
     'dtypes': {'sources': ['dtypes.pyx']},
-    'conversion': {'sources': ['conversion.pyx', 'src/datetime/np_datetime.c'],
-                   'include_dirs': inc_datetime},
-    'fields': {'sources': ['fields.pyx', 'src/datetime/np_datetime.c']},
+    'conversion': {'sources': ['conversion.pyx']},
+    'fields': {'sources': ['fields.pyx']},
     'nattype': {'sources': ['nattype.pyx']},
-    'np_datetime': {'sources': ['np_datetime.pyx', 'src/datetime/np_datetime.c', 'src/datetime/np_datetime_strings.c'],
-                    'include_dirs': inc_datetime},
-    'offsets': {'sources': ['offsets.pyx', 'src/datetime/np_datetime.c'],
-                'include_dirs': inc_datetime},
-    'parsing': {'sources': ['parsing.pyx', '../src/parser/tokenizer.c'],
-                'include_dirs': klib_include},
-    'period': {'sources': ['period.pyx', 'src/datetime/np_datetime.c'],
-                'include_dirs': inc_datetime},
-    'strptime': {'sources': ['strptime.pyx', 'src/datetime/np_datetime.c'],
-                'include_dirs': inc_datetime},
-    'timedeltas': {'sources': ['timedeltas.pyx', 'src/datetime/np_datetime.c'],
-                   'include_dirs': inc_datetime},
-    'timestamps': {'sources': ['timestamps.pyx', 'src/datetime/np_datetime.c'],
-                   'include_dirs': inc_datetime},
-    'timezones': {'sources': ['timezones.pyx', 'src/datetime/np_datetime.c'],
-                  'include_dirs': inc_datetime},
-    'tzconversion': {'sources': ['tzconversion.pyx', 'src/datetime/np_datetime.c'],
-                     'include_dirs': inc_datetime},
-    'vectorized': {'sources': ['vectorized.pyx', 'src/datetime/np_datetime.c'],
-                    'include_dirs': inc_datetime}
+    'np_datetime': {'sources': ['np_datetime.pyx']},
+    'offsets': {'sources': ['offsets.pyx']},
+    'parsing': {'sources': ['parsing.pyx', '../src/parser/tokenizer.c']},
```

Comment:
I'm not sure why this still needs tokenizer.c - should be replaced via the capsule import but likely missed